Makefile

```diff
@@ -72,7 +72,7 @@ endif
 
 ##@ Build
 
-build: generate fmt vet ## Build manager binary.
+build: generate fmt vet golangci-lint kube-linter ## Build manager binary.
```

Comment:
Let's add it to `docker-build` target as well for consistency.

Reply:
Done